backend/app/Http/Controllers/EmployeeController.php

```diff
@@ -220,25 +221,25 @@ public function getEmplyeesByDepartment($departmentId)
 
             return $this->sendResponse(false, null, 'All Department Employees', $employees, Response::HTTP_OK);
         } catch (Exception $e) {
-            return $this->sendResponse(true, 'Employees not fetched', $e->getMessage(), null,  Response::HTTP_INTERNAL_SERVER_ERROR);
+            return $this->sendResponse(true, $e->getMessage(), 'Employees not fetched', null,  Response::HTTP_INTERNAL_SERVER_ERROR);
         }
     }
 
     //this will fetch all employees for admin
     public function getAllEmployees()
     {
         try {
-            $companies = Employee::paginate(10);
-
+            $allEmployees = Employee::paginate(10);
+            $totalCount = $allEmployees->total();
             return $this->sendResponse(
                 false,
                 null,
-                'All employees',
-                $companies,
+                "All employees (count: $totalCount)",
```

Comment:
why are you returning just count here? This belongs to admin... You are not expected to change this.. There's another endpoint doing this

Reply:
That's just a response abi? I only made it more descriptive.
All employees >>>> All employees (count: 3)

No wahala sha. I don change am

Then about the response, this is the format, at least that was what I got from Ben's implementation on authentication controller.
`sendResponse(true, $e->getMessage(), "Something went wrong logging in, please try again", null, 500);`
``bool $errorState = true,
        string $error = null,
        string $message = 'OK',
        $data = null,
        int $code = 200
```
I don't know who decided to interchange some of the responses under my controller, maybe I did so myself but decided to use a uniform response for all. For my try blocks, $error is the exception error [$e->getMessage()], and $message is my own description of the error.